src/app/layouts/default/shared.js

```diff
@@ -84,14 +84,15 @@ function useCampaign(stickyData) {
     }, [start, mode]);
 }
 
-const stickyDataPromise = Promise.all([cmsFetch('sticky/'), cmsFetch('snippets/givebanner')])
```

Comment:
Having it kick this off on file load gave me problems in testing so I moved it into the function that uses it.